pcse_gym/envs/configs/Wofost81_NWLP_MLWB_SNOMIN.conf

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
```

Comment:
Do we want this is a hard copy? Or just use the one from pcse and then modify within cropgym?

Reply:
That's also an option. I don't mind either!